src/org/loklak/api/search/LocationWiseTimeService.java

```diff
@@ -85,8 +85,7 @@ public static SusiThought locationWiseTime(String query) {
 				JSONObject obj = new JSONObject();
 				String l = e.getElementsByTag("a").text();
 				obj.put("location", l);
-				e = e.siblingElements().get(0);
-				String t = e.text();
+				String t = e.nextElementSibling().text();
```

Comment:
Oops. It is the commit of PR https://github.com/loklak/loklak_server/pull/768 added here mistakenly.